rpcudp/protocol.py

```diff
@@ -33,6 +40,7 @@ def __init__(self, wait_timeout=5):
         self.transport = None
 
     def connection_made(self, transport):
+        """callback function that is made on asyncio end."""
```

Comment:
Just a bit more detail / also matches style of other single line comments
```suggestion
        """
        Callback function made by asyncio once a connection has been initiated
        """
```